django_comments/compat.py

```diff
@@ -0,0 +1,6 @@
+import django
+
+if django.VERSION < (1, 7):
+    from django.contrib.sites.models import get_current_site
+else:
+    from django.contrib.sites.shortcuts import get_current_site
```

Comment:
@collinanderson pull request updated, sorry for the confusion, my bad.. thanks!!!!!!!